mypy/build.py

```diff
@@ -337,13 +337,14 @@ def import_priority(imp: ImportBase, toplevel_priority: int) -> int:
     return toplevel_priority
 
 
-def load_custom_plugins(default_plugin: Plugin, options: Options, errors: Errors) -> Plugin:
-    """Load custom plugins if any are configured.
+def load_plugins(options: Options, errors: Errors) -> Plugin:
+    """Load all configured plugins.
 
-    Return a plugin that chains all custom plugins (if any) and falls
-    back to default_plugin.
+    Return a plugin that encapsulates all plugins chained together. Always
+    at least include the default plugin.
```

Comment:
So a user plugin can override the default plugin. Should a hook returned by a user plugin be able to fall back dynamically on the default plugin's hook for the same function? E.g. suppose to add a special case for `open()` I would have to copy the default `open()` handler and modify it. It would be nice if there was some kind of "super call" mechanism for this.